test/hotspot/jtreg/compiler/lib/ir_framework/CompilePhase.java

```diff
@@ -86,6 +86,14 @@ public enum CompilePhase {
     ITER_GVN_AFTER_ELIMINATION(     "Iter GVN after Eliminating Allocations and Locks"),
     BEFORE_PRE_MAIN_POST(           "Before Pre/Main/Post Loops"),
     AFTER_PRE_MAIN_POST(            "After Pre/Main/Post Loops"),
+    BEFORE_POST_LOOP(               "Before Post Loop"),
+    AFTER_POST_LOOP(                "After Post Loop"),
+    BEFORE_REMOVE_EMPTY_LOOP(       "Before Remove Empty Loop"),
+    AFTER_REMOVE_EMPTY_LOOP(        "After Remove Empty Loop"),
+    BEFORE_ONE_ITERATION_LOOP(      "Before Replacing One Iteration Loop"),
+    AFTER_ONE_ITERATION_LOOP(       "After Replacing One Iteration Loop"),
```

Comment:
```suggestion
    BEFORE_ONE_ITERATION_LOOP(      "Before Replacing One-Iteration Loop"),
    AFTER_ONE_ITERATION_LOOP(       "After Replacing One-Iteration Loop"),
```

I see there are also a few more occurrences that I think needs updating (`grep` for "one iteration loop")